### PR TITLE
Fix py3 syntax/import errors on pywebsocket

### PR DESCRIPTION
**Pull request by saschanaz**
#9599 says the wpt clone of pywebsocket is forked so maybe we can do some Py3 patch.

This is a partial fix that does not fix everything.

### CODE REVIEW

File: tools/pywebsocket/mod_pywebsocket/standalone.py [hunk 1]
```diff
@@ -348,8 +348,8 @@ def recv(self, bufsize, flags=0):
 
         try:
             return self._connection.recv(bufsize)
-        except OpenSSL.SSL.SysCallError, (err, message):
```
Comment (Ms2ger) [inline tools/pywebsocket/mod_pywebsocket/standalone.py]: I have been unable to figure out what this syntax meant; would appreciate a second look from someone who knows.

File: tools/pywebsocket/mod_pywebsocket/standalone.py [hunk 2]
```diff
@@ -795,14 +795,14 @@ def parse_request(self):
...
-        except dispatch.DispatchException, e:
+        except dispatch.DispatchException as e:
             self._logger.info('Dispatch failed for error: %s', e)
             self.send_error(e.status)
             return False
 
         # If any Exceptions without except clause setup (including
         # DispatchException) is raised below this point, it will be caught
-        # and logged by WebSocketServer.
+        # and logged by Websocketserver.
```
Comment (Ms2ger) [inline tools/pywebsocket/mod_pywebsocket/standalone.py]: Revert

File: tools/pywebsocket/mod_pywebsocket/util.py [hunk 3]
```diff
@@ -68,7 +68,7 @@ def get_stack_trace():
     TODO: Remove this when we only support Python 2.4 and above.
```
Comment (Ms2ger) [inline tools/pywebsocket/mod_pywebsocket/util.py]: (Hah)

**Review by Ms2ger — APPROVED** [review]
That's a lot of exceptions!

File: tools/pywebsocket/mod_pywebsocket/standalone.py (hunk 1 above)
Comment (saschanaz) [reply tools/pywebsocket/mod_pywebsocket/standalone.py]: Python 2 exceptions are like tuples:

```
Exception(0,1,2,3)[2]
// 2
```

Python 3 does not allow this and requires using `.args`, which gives a tuple.

**Review by Ms2ger — APPROVED** [review]

**Pull request merged**